modules/siddhi-extensions/map/src/test/java/org/wso2/siddhi/extension/map/ToJSONFunctionExtensionTestCase.java

```diff
@@ -162,4 +163,50 @@ public void receive(Event[] inEvents) {
         Assert.assertTrue(eventArrived);
         executionPlanRuntime.shutdown();
     }
+
+    @Test(expected = ExecutionPlanValidationException.class)
+    public void testToJSONFunctionExtension3() throws InterruptedException {
+        log.info("ToJSONFunctionExtension TestCase with test attributeExpressionExecutors length");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        siddhiManager.setExtension("str:concat", ConcatFunctionExtension.class);
+
+        String inStreamDefinition = "\ndefine stream inputStream (symbol string, price long, volume long);";
+        String query = ("@info(name = 'query1') from inputStream select "
+                + "map:createFromJSON(\"{'symbol':'WSO2','price':100,'volume':100,'last5val':{'price':150,'volume':200}}\") as hashMap insert into outputStream;" +
+                "from outputStream " +
+                "select map:toJSON() as jsonString " +
+                "insert into outputStream2");
+
+        ExecutionPlanRuntime executionPlanRuntime = siddhiManager.createExecutionPlanRuntime(inStreamDefinition + query);
+
+
+        InputHandler inputHandler = executionPlanRuntime.getInputHandler("inputStream");
```

Comment:
Below three lines are not needed. createExecutionplanRuntime will throw the exception